### ISSUE TITLE
SHA-1 comparison fails because of different case

### ISSUE THREAD

**Issue by PiotrTrzpil** [cvogt/cbt#227]
The following build file fails:

```
import cbt._
class Build(val context: Context) extends BaseBuild {
  override def dependencies = {
    super.dependencies ++
      Resolver(new java.net.URL("http://maven.spikemark.net/roundeights")).bind(
        "com.roundeights" %% "hasher" % "1.2.0"
      )
  }
}
```

Log:

```
downloading http://maven.spikemark.net/roundeights/com/roundeights/hasher_2.11/1.2.0/hasher_2.11-1.2.0.pom
java.lang.AssertionError: assertion failed: 963B131C79D27F350B80792E28030E18090A3AE0 == 963b131c79d27f350b80792e28030e18090a3ae0
```

**Comment by cvogt** [comment]
thx for reporting, much appreciated :)

**Comment by cvogt** [comment]
fixed in #229
